torch/_dynamo/variables/torch.py

```diff
@@ -494,6 +499,15 @@ def get_state_from_generator():
             return TorchVariable(torch.add, **options).call_function(
                 tx, [args[0], result], {}
             )
+        elif (
+            self.value == torch.distributed.get_process_group_ranks
```

Comment:
I assume we capture into the functional collectives so that we get these two calls and bake the pg -> ranks to the IR?

Reply:
Yes